wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -294,8 +294,8 @@ pub enum WalletCommand {
     #[clap(name = "staking-list-created-block-ids")]
     ListCreatedBlocksIds,
 
-    /// Get the current staking VRF key for this account
-    #[clap(name = "staking-get-vrf-key")]
+    /// Show the issued staking VRF keys for this account
```

Comment:
Show the issued staking VRF (Verifiable Random Function) keys for this account. These keys are generated when pools are created. VRF keys are used as a trustless mechanism to ensure the randomness of the staking process, where no one can control the possible outcomes, to ensure decentralization.